docs/README.md

```diff
@@ -107,15 +107,19 @@ See [Deployment docs](./advanced/deploy.md) for all available options
 
 ## Requirements
 
-[TODO]: on the cluster
-
 Minimal dependencies:
 
-- Git repository (public or private)
-- Kubernetes
-  - giving the kubeconfig to kontinuous to access the project scope
-  - a dedicated ci namespace per project and per cluster containing all needed secrets (optional)
-  - [kube-janitor](https://codeberg.org/hjacobs/kube-janitor) is recommended for automatic resources cleaning
+- **git**: The code to be deployed must be available in a git repository (if the repository is private: TODO describe needed additionnal configuration).
+- **kubernetes**: you must have a correct kubernetes context configured in the shell (either via default kubeconfig or via environnement variable `KUBECONFIG`).
+
+Recommended (optional) dependencies:
+
+- If using **rancher**: either define a `RANCHER_PROJECT_ID` variable or have a dedicated ci namespace per project and per cluster with the projectId specified in an annotation.
+- A namespace containing all the needed secrets to be imported from to the destination namespace (where the kontinuous jobs and deployments will be run).
```

Comment:
```suggestion
- If using **rancher**: define a `RANCHER_PROJECT_ID` variable
```

Il faudra peut-être une section spéciale rancher

Pour les secrets, si je me trompe pas, ca dépend de comment les pipelines sont configurés, donc rien d'obligatoire

Reply:
non rien d'obligatoire, c'est pour ça que c'est dans la section "recommended" :) et yes section dédiée rancher mais en v2 non ?

Reply:
yes recommended mais à ce moment là on peut recommander bcp de choses. ca me parait confusing comme reco "all the needed secrets" à ce moment là de la doc

ok pour préciser pour rancher par la suite, ou mieux, se passer de rancher... :)